moveit_ros/move_group/src/move_group.cpp

```diff
@@ -128,15 +130,15 @@ class MoveGroupExe
 
     // add capabilities listed in ROS parameter
     std::string capability_plugins;
-    if (node_handle_.getParam("capabilities", capability_plugins))
+    if (nh.getParam("capabilities", capability_plugins))
```

Comment:
capabilities and disabled_capabilities do not use the `default_planning_pipeline` namespaces. This breaks specifying custom or disabling standard capabilities.

Reply:
You are right. I was misled by the fact that the Pilz planner loads its specific capabilities from its own namespace.
Works now.